fastlane/Fastfile

```diff
@@ -3,13 +3,42 @@
 # The env variables are contained in an .env.default file.
 
 # Define the minimal Fastlane version
-fastlane_version "1.95.0"
+fastlane_version "2.35.1"
 
 # Use the iOS platform as default
 default_platform :ios
 
+# Default path formatters
+project_xcconfig_path = "%s/ConfigurationFiles/%s.xcconfig"
```

Comment:
Yes, I think this is right. We should match the file system and it's easier without spaces, but just saying don't forget to add to the documentation that this only works for this path and if not you should change it in the Fastfile.